prompt_templates/zero_shot/zero_shot_prompt.py

```diff
@@ -0,0 +1,72 @@
+from pydantic import create_model
+import re
+
+
+class ZeroShotPrompt:
```

Comment:
We may want to define a base class Prompt that is inherited by every specialised prompt class. I imagine there will be overlapping functionalities in all the specialised prompt types that would justify the base class.